tests/virt/utils.py

```
@@ -355,3 +363,112 @@ def validate_machine_type(vm, expected_machine_type):
+    if vm.os_flavor.startswith(OS_FLAVOR_WINDOWS):
+        expected_gpu_name = (
+            supported_gpu_device[VGPU_PRETTY_NAME_STR]
+            if "vgpu" in vm.name
+            else supported_gpu_device[GPU_PRETTY_NAME_STR]
+        )
+        assert expected_gpu_name in get_gpu_device_name_from_windows_vm(vm=vm), (
+            f"GPU device {expected_gpu_name} does not exist in windows vm {vm.name}"
+        )
+    else:
+        assert get_num_gpu_devices_in_rhel_vm(vm=vm) == 1, (
+            f"GPU device {fetch_gpu_device_name_from_vm_instance(vm=vm)} does not exist in rhel vm {vm.name}"
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)